unpackConfigurationsMK.py

```diff
@@ -193,58 +202,13 @@ def unpackConfigurationMK(File,
             d["Btot"] = abs(d["Btot"])
             d["Bpol"] = abs(d["Bpol"])
         
+        # Poloidal distance
         d["Spol"] = np.array(returnll(d["R"], d["Z"]))
-        
-        if log_grid == True:
-        # Create log grid from Xpoint to target in Spol space. Half the res goes above and half
-        # goes below the Xpoint. This is much easier to do after the transformations above
-        # because Spol and Xpoints exist and they're all in the right convention.
-
-            Xpoint = d["Xpoint"]
-            # print("side:",side,d["Spol"][Xpoint+1])
-            dymin = 0.001 # size of cell at the target
-            abovex = np.linspace(d["Spol"][Xpoint], d["Spol"][-1],
-                                 int(np.floor(resolution/2)))
-            belowx = np.logspace(np.log10(dymin), np.log10(d["Spol"][Xpoint]),
-                                 int(np.ceil(resolution/2)))
-            belowx = np.insert(belowx, 0, 0) # logspace won't go to 0. Ensure there is one
-            belowx = np.delete(belowx, -1) # Delete duplicate point shared with abovex
-            path_grid_log = np.concatenate([belowx, abovex])
-            
-            # STILL OKAY AT THIS POINT
-
-            # Reinterpolate the grid and redo all the transformations
-            for param in full.keys():
-                loginterp = interpolate.interp1d(d["Spol"], d[param])
-                d[param] = loginterp(path_grid_log)
-     
-            d["Spol"] = path_grid_log
-            d["Xpoint"] = len(belowx)
-            
-        
-        # Assemble all the output variables
-        d["Bx"] = d["Btot"][d["Xpoint"]]
-        d["zl"] = np.array(returnzl(d["R"], d["Z"], d["Bx"], np.absolute(d["Bpol"])))
-        d["zx"] = d["zl"][d["Xpoint"]]   
         d["S"] = np.array(returnS(d["R"], d["Z"], d["Btot"], d["Bpol"]))
-        d["Sx"] = d["S"][d["Xpoint"]]
-        d["Spolx"] = d["Spol"][d["Xpoint"]]
-
-        # Full arrays of R and Z
-        d["R_full"] = Rs
-        d["Z_full"] = Zs
-        d["R_ring"] = Rs[sep]
-        d["Z_ring"] = Zs[sep]
         
-
-    """------OUTPUT"""    
-    
-    # Output by geometry type
-    if Type != "box":
-        return data[Type]
-    else:
-        print("Slab geometry not supported yet")
-
+        # Z space distance (combined parallel and flux expansion, see Lipschultz 2016)
+        d["zl"] = np.array(returnzl(d["R"], d["Z"], d["Btot"][d["Xpoint"]], np.absolute(d["Bpol"])))
```

Comment:
is zl very useful? Should we remove

Reply:
Only if someone wants to go back to the origins of the model I guess... I think Fabio was still doing stuff in terms of zl